src/services/technicalRecord.service.ts

```diff
@@ -0,0 +1,65 @@
+import { NewPlateRequest } from '../models/Request.model';
+import { Plates } from '../models/Plates.model';
+import { getSerialNumber } from './serialNumber.service';
+import { TechRecord, Vehicle } from '../models/Vehicle.model';
+import { put } from './dynamodb.service';
+import logger from '../observability/logger';
+
+export const addNewPlate = async (request: NewPlateRequest): Promise<TechRecord[]> => {
+  const techRecords = request.techRecord as TechRecord[];
+
+  if (request.reasonForCreation && request.vtmUsername && techRecords) {
```

Comment:
It would be clearer if you invert this and it would omit the nesting.
```suggestion
  if (!request.reasonForCreation || !request.vtmUsername || !techRecords) {
    throw new Error('Bad Request');
  }
  
  // Carry out business logic...
```

Reply:
Yeah this is a great shout!